samples/apim-policy/README.md

```diff
@@ -0,0 +1,108 @@
+<!-- markdownlint-disable MD024 -->
+# Azure API Management Policy
+
+This folder contains a sample integration of the [Carbon Aware SDK](https://github.com/Green-Software-Foundation/carbon-aware-sdk)
+with [Azure API Management](https://azure.microsoft.com/en-us/products/api-management/) (APIM)
+[policies](https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-policies).
```

Comment:
I would maybe add one more sentence saying how this sample just includes the policies and is not a full integration guide.

Reply:
Good suggestion. Updated first paragraph to clarify what sample covers.